app/src/main/java/com/yogeshpaliyal/deepr/ui/screens/home/DeeprItemSwipable.kt

```
@@ -32,18 +37,42 @@ fun DeeprItemSwipable(
     val dismissState =
         rememberSwipeToDismissBoxState(
             initialValue = SwipeToDismissBoxValue.Settled,
-            positionalThreshold = SwipeToDismissBoxDefaults.positionalThreshold,
+            confirmValueChange = { newValue ->
+                if (newValue != SwipeToDismissBoxValue.Settled) {
+                    val state = dismissStateHolder.value
+                    if (state != null && itemWidth > 0) {
+                        val offset =
+                            try {
+                                abs(state.requireOffset())
+                            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)